src/schemas/json/license-report-config.json

```diff
@@ -75,14 +76,18 @@
       "type": "array"
     },
     "comment": {
-      "description": "export deps. or deps/dev-/opt-/peer- deps. falsey -> output everything",
+      "description": "export deps. or deps/dev-/opt-/peer- deps. falsy -> output everything",
```

Comment:
I would just keep the original spelling as it seems to be acceptable too (I am no native speaker so my "wisdom" comes from [mdn](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)

Reply:
Rollbacked to `falsey`.